internal/database/connections/live/runner.go

```diff
@@ -18,6 +19,7 @@ func RunnerFromDSNs(out *output.Output, logger log.Logger, dsns map[string]strin
 }
 
 func RunnerFromDSNsWithSchemas(out *output.Output, logger log.Logger, dsns map[string]string, appName string, newStore StoreFactory, availableSchemas []*schemas.Schema) (*runner.Runner, error) {
+	var verbose = os.Getenv("SRC_LOG_LEVEL") == "dbug" || os.Getenv("SRC_LOG_LEVEL") == "info"
```

Comment:
```suggestion
	var verbose = env.LogLevel == "dbug" || env.LogLevel == "info"
```
youll need to import `"github.com/sourcegraph/sourcegraph/internal/env"` for this